tests/data/q_date_in_lifetime/general/in_allergyintolerance.ndjson

```diff
@@ -0,0 +1,16 @@
+{"resourceType": "AllergyIntolerance", "recordedDate": "2000-01-01", "id": "valid-recorded", "patient": {"reference": "Patient/dead"}}
```

Comment:
You went pretty hard on the test suite - maybe I should have poked my head in here earlier - as you may be able to see in other test suites, I have not been quite as exhaustive as you - which isn't bad on your part at all. Just maybe saying that my expectations were lower - like, I think it's reasonable to maybe have one resource (allergy intolerance, say) that you go ham on with all the weird edge conditions, then spot check a few other instances of other resource fields to just confirm that, yes you are hitting all the fields in DATES - but I myself would trust that DATES is well tested elsewhere, and we don't need to entirely re-test it here by checking every possible field.

But now that you already did it, it's nice. 😄